### ISSUE TITLE
关于模型中的输入

### ISSUE THREAD

**Issue by zeng9t** [gaussic/text-classification-cnn-rnn#57]
你好，请问模型部分中的第三个placeholder输入keep_prob是什么参数？

**Comment by gaussic** [comment]
https://www.tensorflow.org/api_docs/python/tf/contrib/layers/dropout

```
tf.contrib.layers.dropout(
    inputs,
    keep_prob=0.5,
    noise_shape=None,
    is_training=True,
    outputs_collections=None,
    scope=None,
    seed=None
)
```

Defined in tensorflow/contrib/layers/python/layers/layers.py.

Returns a dropout op applied to the input.

With probability keep_prob, outputs the input element scaled up by 1 / keep_prob, otherwise outputs 0. The scaling is so that the expected sum is unchanged.

### Args:

- inputs: The tensor to pass to the nn.dropout op.
- keep_prob: A scalar Tensor with the same type as x. The probability that each element is kept.
- noise_shape: A 1-D Tensor of type int32, representing the shape for randomly generated keep/drop flags.
- is_training: A bool Tensor indicating whether or not the model is in training mode. If so, dropout is applied and values scaled. Otherwise, inputs is returned.
- outputs_collections: Collection to add the outputs.
- scope: Optional scope for name_scope.
- seed: A Python integer. Used to create random seeds. See tf.set_random_seed for behavior.

### Returns:
A tensor representing the output of the operation.
